core/templates/components/skiplinks/skiplinks.twig

```diff
@@ -0,0 +1,9 @@
+{% set skipID = "skip-to-content-" ~ random() %}
+
+<header>
+  <a href="#{{ anchor|default(skipID) }}" class="su-skiplinks">{{ text|default('Skip to content') }}</a>
+</header>
+
+<main>
+  <h2 id="{{ skipID }}" class="su-sr-only-text" tabindex="-1">{{ skipped_to_title }}</h2>
```

Comment:
The id of the target element should match the href of the skip link, i.e. `#{{ anchor|default(skipID) }}`.

Or `#{{ target_id|default(skipID) }}` if you accept my suggestion on the `.json` file.

Reply:
Added!

Reply:
Thank you!